packages/server/src/views/styles.ts

```diff
@@ -11,7 +11,7 @@ export const getCSSUrl = async () => {
 };
 
 const getCssElementAsString = async () => {
-    if (env.NODE_ENV === "development" && !env.HAS_EXTERNAL_DEV_CONSUMER) {
```

Comment:
Bruker denne ved lokal kjøring av dekoratøren integrert med andre lokale apper. Skal hindre feil som oppstår når `styles`-elementet er tomt. Mulig det kan løses på en annen måte, kanskje vi heller kan sette inn et dummy element i dev-modus, istedenfor en tom string?

Reply:
Kan du bruke samme metode som vi gjør i Playwright: `bun run --cwd packages/server serve-local`?

https://github.com/navikt/decorator-next/blob/main/playwright.config.ts#L74

Reply:
Er av og til kjekt å kunne kjøre den i development modus med HMR osv, samtidig som den er integrert i en annen app lokalt.

Reply:
(Dette vil ikke være noe problem lengre når appene går over til å bruke `/ssr` - soon™️ )

Reply:
Denne funka for meg:

`return "<!-- non-empty css response to make nav-dekoratoren-moduler happy -->";`

Reply:
Hva synes du om en sånn løsning?

Reply:
Det burde funke 👍